test/run_test.py

```diff
@@ -834,11 +834,11 @@ def parse_args():
         help='exclude tests that are run for a specific jit config'
     )
     parser.add_argument(
-        '--run-specified-test-cases',
-        nargs='?',
+        '--use-specified-test-cases',
         type=str,
-        const=SPECIFIED_TEST_CASES_FILE,
-        help='runs specified test cases from previous OSS CI stats from a file, format CSV',
+        choices=['include', 'bring-to-front'],
```

Comment:
I think the 'include' is a little misleading, I think it suggests we run everything and we make sure to include this subset of tests, whereas the actual functionality would be to _only_ run these tests, right? Maybe something like include-only or like, exclude-others or something. 

I also think we should document what these choices mean in the help.

Reply:
i see your concern. `-i` and `--include` option is clearly indicating it is only running those. but not at this option. I can improve the docstring